strands_movebase/strands_movebase_params/costmap_common_params.yaml

```diff
@@ -3,17 +3,32 @@ robot_base_frame: /base_link
 
 map_type: voxel
 
-footprint: [[-0.30,-0.32], # Back side right
-            [-0.52,-0.11], # Back right
-            [-0.52, 0.11], # Back left
-            [-0.30, 0.32], # Back side left
-            [-0.05, 0.31], # Front side left
-            [ 0.05, 0.25], # Thick front left
-            [ 0.15, 0.13], # Thin front left
-            [ 0.20, 0.00], # Front
-            [ 0.15,-0.13], # Thin front right
-            [ 0.05,-0.25], # Thick front right
-            [-0.05,-0.31]] # Front side right
+footprint: [[-0.211159,-0.295000],
```

Comment:
ah! good catch. It's surely because of the approximation tool. I worry about getting much more vertices in because of the dwa control loop becoming too slow.

I wonder what impact this has. Maybe it can even help when you have goals exactly behind the robot, as now the non-symmetry might make it easier for dwa to choose which direction to turn?